int32 room_id
int32 camera_id
int32[] room_to_submap
float32 room_area
RoomModel room_model
